language/stdlib/modules/LibraSystem.move

```diff
@@ -158,6 +159,11 @@ module LibraSystem {
         operator_account: &signer,
         validator_address: address,
     ) acquires CapabilityHolder {
+        assert(
+            Roles::has_validator_operator_role(operator_account) ||
+            Roles::has_validator_role(operator_account),
```

Comment:
I think this is always supposed to be an operator rather than the validator itself, but I could be wrong. But @valerini will know for sure!

Reply:
I think Lera is out this week, cc @davidiw who might know as well?

Reply:
Yeah, we don't envision the case for a validator to be able to reconfigure since only the validator operator should be able to modify the validator config (outside of setting the operator)

Reply:
Awesome! I'll update :)